### ISSUE TITLE
PIO unified debugger

### ISSUE THREAD

**Issue by tablatronix** [gepd/Deviot#174]
Do you know about this?

https://community.platformio.org/t/pio-unified-debugger/1704

Might be useful for something

**Comment by gepd** [comment]
Yes, I've it in my list too, depending on the time I clould add this in `v2.3` or maybe in `v2.2`
